SLObjectiveCRuntimeAdditions/SLObjectiveCRuntimeAdditions.m

```diff
@@ -48,8 +48,8 @@ - (BOOL)implementInstanceMethodNamed:(SEL)selector types:(const char *)types imp
     NSParameterAssert(blockImplementation);
     NSParameterAssert(types);
 
-    __assert_unused SLBlockDescription *blockDescription = [[SLBlockDescription alloc] initWithBlock:blockImplementation];
-    __assert_unused NSMethodSignature *methodSignature = [NSMethodSignature signatureWithObjCTypes:types];
+    __unused SLBlockDescription *blockDescription = [[SLBlockDescription alloc] initWithBlock:blockImplementation];
+    __unused NSMethodSignature *methodSignature = [NSMethodSignature signatureWithObjCTypes:types];
```

Comment:
It is defined [here](https://github.com/OliverLetterer/SLObjectiveCRuntimeAdditions/blob/master/SLObjectiveCRuntimeAdditions.podspec#L15)